src/Authentication/Actions/Email2FA.php

```diff
@@ -109,8 +109,6 @@ public function verify(IncomingRequest $request)
         session()->remove('auth_action');
 
         // Get our login redirect url
-        $loginController = new LoginController();
-
-        return redirect()->to($loginController->getLoginRedirect());
+        return redirect()->to(AuthConfig::loginRedirect());
```

Comment:
You have this hard-coded to the Shield version of the config. I think this needs to use `config()` to ensure it can access the overridden version that will often be in `app/Config`. I think this might necessitate it being a non-static method, also.

```suggestion
        return redirect()->to(config('Auth')->loginRedirect());
```

Reply:
Oh, thanks. Indeed.

Reply:
Fixed.

Reply:
Yes glad this was updated already but Config methods should always be on the instance since they could come from who-knows-where in the wild land of Config discovery 😅